apps/condo/domains/acquiring/components/payments/PaymentsTable.tsx

```diff
@@ -81,20 +82,17 @@ const PaymentsSumInfo: React.FC<IPaymentsSumInfoProps> = ({
     )
 }
 
-function getSumPerPeriod (objs) {
-    let sumAllPayments = 0,
-        sumDonePayments = 0,
-        sumWithdrawnPayments = 0
-
-    for (const key in objs) {
-        sumAllPayments += Number(objs[key].amount)
-        if (objs[key].status === PAYMENT_DONE_STATUS)
-            sumDonePayments += Number(objs[key].amount)
-        else
-            sumWithdrawnPayments += Number(objs[key].amount)
-    }
-
-    return { sumAllPayments, sumDonePayments, sumWithdrawnPayments }
+function GetSum (whereQuery, paymentStatus) {
+    const { data, loading } = useQuery(SUM_PAYMENTS_MUTATION, {
+        fetchPolicy: 'network-only',
+        variables: {
+            where: {
+                ...whereQuery,
+                status: paymentStatus,
+            },
+        },
+    })
+    return loading ? 0 : Number(data.result.sum)
```

Comment:
why we show 0 if the value is still loading?

Reply:
We need to show some value during loading. This value should be number for total sum correct work. It is shows 0 during payments loading, so this behavior is seems correct

Reply:
Just picture yourself in a position of the user for a moment: you open up payments and see 0 in total payments sum :-) This is not OK

This is OK only if the table with the payments below is loading too, and these states are somewhat syncronized.

I'd rather show a skeleton or a loader instead of 0.